src/sentry/integrations/jira/webhooks/descriptor.py

```diff
@@ -23,7 +23,9 @@ class JiraDescriptorEndpoint(Endpoint):
     permission_classes = ()
 
     def get(self, request: Request) -> Response:
-        sentry_logo = absolute_uri(get_asset_url("sentry", "images/logos/logo-sentry.svg"))
+        sentry_logo = absolute_uri(
+            get_frontend_app_asset_url("sentry", "entrypoints/logo-sentry.svg")
```

Comment:
This new URL is where the logo is copied to as a result of this [PR](https://github.com/getsentry/sentry/pull/41398)